app.rb

```diff
@@ -50,4 +50,43 @@ def debug_params
     list.save!
   end
   redirect back
-end
\ No newline at end of file
+end
+
+#----------------- Add New List ------------------
+
+post "/newlist/name" do
+  new_list = List.new("#{params[:id]}")
+  new_list.name = "#{params[:id]}" 
+  new_list.save!
+  List.new("#{new_list.name}")
+  #new_list.filename
+  redirect back
+end
+
+
+#----------------- Delete a List ------------------
+
+post "/delist/name" do
+  list_dump = List.new("#{params[:id]}")
+  target = list_dump.id.concat(".md")
+  files = Dir["data/*.md"]
+  files.each do |file|
+    file.delete("target")
+  end  
+  redirect back
+end
+
+
+
+#----------------Time Setter--------------------
+
+time = Time.new()
+@today = "#{time.day}/#{time.month}/#{time.year}"
+
+
+
+
+
+
+
+
```

Comment:
remove all the empty lines. in general dont have more than 1 empty line in a row